css/css-grid_image-gallery/css/styles.css

```
@@ -41,6 +40,9 @@ body {
 .teaser-grid {
   min-width: 40rem;
   min-height: 100vh;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(2, 1fr);
```

rows are should be half of column's height as per design, below line worked for me

```suggestion
  grid-template-rows: 25% 25%;
```